wp_insert_rows.php

```diff
@@ -17,18 +18,21 @@
 	*
 	*  }
 	*
-	*  wp_insert_rows($insert_arrays);
+	*  wp_insert_rows($insert_arrays, $wpdb->tablename, true, "column_name_of_Primary_key");
 	*
 	*
 	* @param array $row_arrays
 	* @param string $wp_table_name
+	* @param boolean $update
+	* @param string $prikeyname
 	* @return false|int
 	*
 	* @author	Ugur Mirza ZEYREK
+	* @contributor Travis Grenell
 	* @source http://stackoverflow.com/a/12374838/1194797
 	*/
 	 
-	function wp_insert_rows($row_arrays = array(), $wp_table_name) {
+function wp_insert_rows($row_arrays = array(), $wp_table_name, $update, $prikeyname) {
```

Comment:
An existing user should not have to modify old codes. Therefore, I think adding default values as "$update=false, $primary_key = null" should be the way it supposed to be.

BTW, I'm very glad to see your contribution on this repo :)